examples/Collada.js

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
-require(['./WorldWindShim', './LayerManager'], function (WorldWind, LayerManager) {
+require(['./WorldWindShim', './LayerManager'],
```

Comment:
Good catch. If this is going to be modified, we should maintain our usual indentation, e.g:
````javascript
require(['./WorldWindShim',
         './LayerManager'],
     function (WorldWind,
               LayerManager) {
````